lms/templates/footer-edx-v3.html

```diff
@@ -1,90 +1,76 @@
+            % endfor
         </nav>
-        <p class="copyright">
-            ## Using "edX Inc." explicitly here for copyright purposes (settings.PLATFORM_NAME is just "edX", and this footer is only used on edx.org)
-            ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
-            <p>&copy; ${settings.COPYRIGHT_YEAR} edX Inc. All rights reserved except where noted. EdX, Open edX and the edX and Open edX logos are registered trademarks of trademarks of edX Inc.</p>
-        </p>
+        <p class="copyright">${footer['copyright']}</p>
+
+        ## The OpenEdX link may be hidden when this view is served
+        ## through an API to partner sites (such as marketing sites or blogs),
+        ## which are not technically powered by OpenEdX.
+        % if not hide_openedx_link:
         <div class="openedx-link">
-          <a href="http://open.edx.org" title="${_("Powered by Open edX")}">
-            <img alt="${_("Powered by Open edX")}" src="https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png">
+          <a href="${footer['openedx_link']['url']}" title="${footer['openedx_link']['title']}">
+            <img alt="${footer['openedx_link']['title']}" src="${footer['openedx_link']['image']}" width="140">
           </a>
         </div>
+        % endif
     </div>
 
     <div class="external-links">
-        <div class="social-media-links">
-          ## Translators: This is the website name of www.facebook.com. Please
-          ## translate this the way that Facebook advertises in your language.
-          <a href="${settings.SOCIAL_MEDIA_FOOTER_URLS.get('facebook', '#')}" class="sm-link external" title="${_("Facebook")}" rel="noreferrer">
-            <span class="icon fa fa-facebook-square element-invisible"></span>
-          </a>
-
-          ## Translators: This is the website name of www.twitter.com. Please
-          ## translate this the way that Twitter advertises in your language.
-          <a href="${settings.SOCIAL_MEDIA_FOOTER_URLS.get('twitter', '#')}" class="sm-link external" title="${_("Twitter")}" rel="noreferrer">
-            <span class="icon fa fa-twitter element-invisible"></span>
-          </a>
-
-          ## Translators: This is the website name of www.linked.com. Please
-          ## translate this the way that LinkedIn advertises in your language.
-          <a href="${settings.SOCIAL_MEDIA_FOOTER_URLS.get('linkedin', '#')}" class="sm-link external" title="${_("LinkedIn")}" rel="noreferrer">
-            <span class="icon fa fa-linkedin-square element-invisible"></span>
-          </a>
+      <div class="social-media-links">
+        % for link in footer['social_links']:
+        <a href="${link['url']}" class="sm-link external" title="${link['title']}" rel="noreferrer">
+          <span class="icon fa ${link['icon-class']} element-invisible"></span>
```

Comment:
Sorry @wedaly, this was my mistake. Can you please remove element-invisible and set aria-hidden="true" instead?